Composer/packages/client/src/pages/language-generation/table-view.tsx

```diff
@@ -151,8 +151,8 @@ const TableView: React.FC<TableViewProps> = props => {
 
       // build usedIn map
       templates.forEach(template => {
-        templateUsedInDialogMap[template.Name] = dialogs.reduce<string[]>((result, dialog) => {
-          if (dialog.lgTemplates.includes(template.Name)) {
+        templateUsedInDialogMap[template.name] = dialogs.reduce<string[]>((result, dialog) => {
```

Comment:
```suggestion
        templateUsedInDialogMap[name] = dialogs.reduce<string[]>((result, dialog) => {
```

Reply:
Next time when you are writing this logic, you can also consider writing it this way to make it easier to read.

```js
templateUsedInDialogMap[name] = dialogs.filter(dialog => dialog.lgTemplates.includes(name)).map(dialog => dialog.id);
```